tests/main/confinement-classic/task.yaml

```diff
@@ -1,10 +1,13 @@
 summary: trivial snap with classic confinement runs correctly
+
+systems: [-ubuntu-core-16-*, -fedora-*]
```

Comment:
We have systems here and also some lines below in the same file. Copy/paste error here maybe?